### ISSUE TITLE
lotus-miner sectors expired list a sector not expired yet. 

### ISSUE THREAD

**Issue by s0nik42** [filecoin-project/lotus#7911]
### Checklist

- [X] This is **not** a security-related bug/issue. If it is, please follow please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).
- [X] This is **not** a question or a support request. If you have any lotus related questions, please ask in the [lotus forum](https://github.com/filecoin-project/lotus/discussions).
- [X] This is **not** a new feature request. If it is, please file a [feature request](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Ffeature&template=feature_request.yml) instead.
- [X] This is **not** an enhancement request. If it is, please file a [improvement suggestion](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Fenhancement&template=enhancement.yml) instead.
- [X] I **have** searched on the [issue tracker](https://github.com/filecoin-project/lotus/issues) and the [lotus forum](https://github.com/filecoin-project/lotus/discussions), and there is no existing related issue or discussion.
- [X] I am running the [`Latest release`](https://github.com/filecoin-project/lotus/releases), or the most recent RC(release canadiate) for the upcoming release or the dev branch(master), or have an issue updating to any of these.
- [X] I did not make any code changes to lotus.

### Lotus component

- [ ] lotus daemon - chain sync
- [ ] lotus miner - mining and block production
- [X] lotus miner/worker - sealing
- [ ] lotus miner - proving(WindowPoSt)
- [ ] lotus miner/market - storage deal
- [ ] lotus miner/market - retrieval deal
- [ ] lotus miner/market - data transfer
- [ ] lotus client
- [ ] lotus JSON-RPC API
- [ ] lotus message management (mpool)
- [ ] Other

### Lotus Version

```text
Daemon:  1.13.0+mainnet+git.7a55e8e89+api1.4.0
Local: lotus version 1.13.0+mainnet+git.7a55e8e89
```

### Describe the Bug

 lotus-miner sectors expired is listing sector 0 `which is not expired yet.` 
Note : this miner has the double sector 0 bug as well could be related. 

```
 lotus-miner sectors expired
Sector	State	Expiration
0	Proving	1566516 (in 5 weeks 5 days)
1453	Proving	1330356 (5 weeks 6 days ago)
1562	Proving	1359156 (4 weeks 3 days ago)
1563	Proving	1359156 (4 weeks 3 days ago)
1564	Proving	1359156 (4 weeks 3 days ago)
```

### Logging Information

```text
miner sectors list  |head
ID    State       OnChain  Active  Expiration                         Deals  DealWeight  VerifiedPower
0     Proving     YES      NO      1566516 (in 5 weeks 5 days)        CC
0     Proving     YES      NO      1566516 (in 5 weeks 5 days)        CC
1     Proving     YES      YES     1566516 (in 5 weeks 5 days)        2      116KiB
18    Proving     YES      YES     1573498 (in 6 weeks 22 hours)      3      58.27MiB
19    Proving     YES      YES     1575129 (in 6 weeks 1 day)         4      116.4MiB
20    Proving     YES      YES     1573498 (in 6 weeks 22 hours)      5      174.4MiB
21    Proving     YES      YES     1573498 (in 6 weeks 22 hours)      6      175.1MiB
```
```


### Repo Steps

_No response_

**Comment by Reiers** [comment]
Hi @s0nik42 

Thanks for taking the time to create this report.

We are aware of the double sector 0 issue https://github.com/filecoin-project/lotus/issues/7779

In this case, both those sectors where `mark-for-upgrade` and or no longer active. Therefor its safe to remove them.
It will remove both sectors (0) when you run the cmd:
`lotus-miner sectors expired --remove-expired
`

See this Slack thread: https://filecoinproject.slack.com/archives/CPFTWMY7N/p1638564279278000 , if you have any doubts.

Closing the issue. Since we already got a issue open for the double 0 sectors. Thanks again !
